amundsen_application/static/js/ducks/tableMetadata/reducer.ts

```diff
@@ -328,6 +427,29 @@ export default function reducer(
         ...state,
         tableOwners: tableOwnersReducer(state.tableOwners, action),
       };
+    case GetTableLineage.SUCCESS:
+    case GetTableLineage.FAILURE:
+      return {
+        ...state,
+        tableLineage: {
+          lineage: (<GetTableLineageResponse>action).payload.lineage,
+          status: (<GetTableLineageResponse>action).payload.status,
+        },
+      };
+    case GetColumnLineage.SUCCESS:
+    case GetColumnLineage.FAILURE:
+      const columnLineage = (<GetColumnLineageResponse>action).payload.lineage;
+      const { columnName } = (<GetColumnLineageResponse>action).payload;
```

Comment:
You might combine these two in one?
```suggestion
      const { columnName, columnLineage: lineage } = (<GetColumnLineageResponse>action).payload;
```

Reply:
Haven't used this before. Thanks